doc/gov/chapters/chapter04.md

```diff
@@ -34,7 +34,7 @@ For each Workstream, following leadership positions exist:
 <a name="4.2.1"></a>
 ### 4.2.1 Length of Terms
 
-The length of term for all CNTT leadership roles is **12** calender months. It will start typically in February each year and end around the same time the year after. (Exact dates and timing might vary to accomodate CNTT releases and F2F workshops).
+The length of term for all CNTT leadership roles is **12** calender months. It will start typically in February each year and end around the same time the year after. (Exact dates and timing might vary to accomodate CNTT releases and workshops).
```

Comment:
```suggestion
The length of term for all CNTT leadership roles is **12** calendar months. It will start typically in February each year and end around the same time the year after.  The exact dates and timing might vary to accommodate CNTT releases and workshops).
```